subsys/bluetooth/mesh/light_ctrl_srv.c

```diff
@@ -1310,7 +1311,13 @@ static int light_ctrl_srv_start(struct bt_mesh_model *mod)
 
 	switch (srv->lightness->ponoff.on_power_up) {
 	case BT_MESH_ON_POWER_UP_OFF:
-		ctrl_disable(srv);
+		if (atomic_test_bit(&srv->flags,
+			FLAG_CTRL_SRV_MANUALY_ENABLED)) {
+			ctrl_enable(srv);
+		} else {
+			ctrl_disable(srv);
+		}
```

Comment:
I don't think you need `ctrl_disable()` for these two, they've never been enabled.

Also, I think you'll also end up starting the movement in the enable call with this. I'd suggest adding a flag called `STARTED` or something, and enable it at the start of this function. Also, protect `transition_start` and `reg_start` in `ctrl_enable` with it, to prevent the regulator and state machine from starting before the model is started.

Reply:
Resolved. We came to an agreement in person.